DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Administrator/AdministratorController.cs

```diff
@@ -138,5 +165,36 @@ private IEnumerable<string> GetCourseCategories(int centreId)
             categories = categories.Prepend("All");
             return categories;
         }
+
+        private bool NewUserRolesExceedAvailableSpots(EditRolesViewModel newRoles, AdminUser oldUserDetails)

```

Comment:
I kept this check method in the controller, rather than making it part of a service method, as I was under the impression that Super Admins will be able to add to roles even when they are full. In which case it would make more sense not to bake this check in to the update procedure if it will eventually be side-stepped anyway. So the change is certainly feasible, and I can easily make it, but I just want to clarify if we think it is best?

Reply:
We can always add a flag to skip the check as a param to the service method